### PR TITLE
QR Code Generation Script

### PR DESCRIPTION
**Pull request by suhasasumukh**
Changes made:

- Made the logo path a parameter with a default value
- Combined the logo resizing logic to improve readability
- Used os.path.join to create the logo path.

### CODE REVIEW

**Comment by GitBolt** [comment]
Did you try running it? Seems like os.path.join part is removed and the readability doesn't seem to be effected that much with the changes

**Comment by RevTpark** [comment]
btw, the solana pay docs itself doesn't allow changing the logo path since it represents it is made by Solana Pay.

File: solathon/solana_pay/create_qr.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 from qrcode.image.styles.moduledrawers.pil import RoundedModuleDrawer
 import os
 
-def create_qr(link: str, size: int = 10, background: str = 'white', color: str = 'black', border: int = 2) -> BytesIO:
+def create_qr(link: str, size: int = 10, background: str = 'white', color: str = 'black', border: int = 2, logo_path: str = "qr-logo.png") -> BytesIO:
```
Comment (RevTpark) [inline solathon/solana_pay/create_qr.py]: We cant let users add their own logo right now
refer `https://docs.solanapay.com/api/core/function/createQR`

Comment (suhasasumukh) [reply solathon/solana_pay/create_qr.py]: oh I have resolved it

**Comment by suhasasumukh** [comment]
> Did you try running it? Seems like os.path.join part is removed and the readability doesn't seem to be effected that much with the changes

it must be fixed now

File: solathon/solana_pay/create_qr.py [hunk 2]
```diff
@@ -14,6 +14,7 @@ def create_qr(link: str, size: int = 10, background: str = 'white', color: str =
         background (str): The background color of the QR code.
         color (str): The color of the QR code.
         border (int): The border of the QR code.
+        logo_path (str): The path to the fixed logo image.
```
Comment (RevTpark) [inline solathon/solana_pay/create_qr.py]: Not a param anymore

**Review by RevTpark — CHANGES_REQUESTED** [review]

File: solathon/solana_pay/create_qr.py (hunk 2 above)
Comment (suhasasumukh) [reply solathon/solana_pay/create_qr.py]: @RevTpark fixed

**Comment by suhasasumukh** [comment]
@GitBolt it should be working fine now

**Comment by GitBolt** [comment]
Well, there were plenty of commits there just to have the changes introduced to be reversed back. The formatting is a very minor change left only. Merging won't make sense here.

**Pull request closed without merging**